development/compiling/compiling_for_android.rst

```diff
@@ -94,8 +94,8 @@ root directory with the following arguments:
 
 ::
 
-    scons platform=android target=release android_arch=armv7
-    scons platform=android target=release android_arch=arm64v8
+    scons platform=android target=template_release android_arch=armv7
```

Comment:
`android_arch` no longer exist, it's `arch` now (on all platforms).

Reply:
updated, thanks